### ISSUE TITLE
[BUG] Storybook Accessibility Keyboard do not let you everywhere

### ISSUE THREAD

**Issue by MaciejWolowski** [vuestorefront/storefront-ui#2406]
**Describe the bug**
Testing accessibility. Using keyboard shortcuts (tab, arrows, enter) you cannot enter the folders highlighted in the screenshot. And go through what is inside.

**How to reproduce**

1. Go to Storybook
1. Using your keyboard "Tab" and "Arrows" go to every highlighted folder
1. Press "Enter"

**Expected behavior**
Folder should roll open and you should be able to open options from inside

**Actual behavior**
Nothing happens.

**Screenshots**
![Storybook Accessibility Keyboard do not let you everywhere](https://user-images.githubusercontent.com/103260041/168227954-be06f45f-2929-499b-9fc7-a4e7312a34d6.png)

**Devices (please complete the following information):**

* Device: [Desktop]
* OS: [Windows 11]
* Browser: [Chrome]

┆Issue is synchronized with this [Jira Zadanie](https://vsf.atlassian.net/browse/SFUI-145) by [Unito](https://www.unito.io)

**Comment by AdamPawlinski** [comment]
@MaciejWolowski Could you please specify what is wrong here? I was able to open all folders that you pointed. Maybe it's the question of system? 
![storybook_sidebar_keyboard](https://user-images.githubusercontent.com/32042425/172132843-a496dc07-f80d-477b-913c-940e1ea09afc.gif)
